docs/development/ol-proxy.md

```diff
@@ -52,6 +52,6 @@ Once the message is sent to the proxy, you will see the OpenLineage message cont
 Not only the OpenLineage proxy is useful in receiving the monitoring the OpenLineage events, it can also be used to relay the events to other endpoints. Please see the [example](https://github.com/OpenLineage/OpenLineage/blob/main/proxy/proxy.example.yml) of how to set the proxy to relay the events via Kafka topic or HTTP endpoint.
```

Comment:
```suggestion
Not only the OpenLineage proxy is useful in receiving the monitoring the OpenLineage events, it can also be used to relay the events to other endpoints. Please see the [example](https://github.com/OpenLineage/OpenLineage/blob/main/proxy/backend/proxy.example.yml) of how to set the proxy to relay the events via Kafka topic or HTTP endpoint.
```